docs/resources/Emoji.md

```diff
@@ -79,15 +79,15 @@ Emojis cannot be converted between normal and premium after creation.
 
 ## List Guild Emojis % GET /guilds/{guild.id#DOCS_RESOURCES_GUILD/guild-object}/emojis
 
-Returns a list of [emoji](#DOCS_RESOURCES_EMOJI/emoji-object) objects for the given guild.
+Returns a list of [emoji](#DOCS_RESOURCES_EMOJI/emoji-object) objects for the given guild. Includes `user` fields if the bot has the `CREATE_GUILD_EXPRESSIONS` or `MANAGE_GUILD_EXPRESSIONS` permission.
```

Comment:
This should not `user` fields for bots regardless if they have permissions or not.